docs/docs/testing/e2e/setup.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Setup
+tags:
+  - testing
+  - e2e
+  - playwright
+  - infrastructure
+---
+
+## Overview
+
+This document explains at a high level how we create our app instance, run the e2e tests and what technology we're using for e2e tests. As well as a small section on writing tests.
+
+## Get Started
+
+Because we require a "fresh" instance to assert our e2e tests against this is included in the testing script so all you need to run is:
+
+```shell
+yarn test:e2e
```

Comment:
Is this still up-to-date? I have the impression it would still be a two-step setup using `test:generate-app` followed by `test:e2e`?

Reply:
i'll double check the whole document, but it's only one command.